docs/Installation/Installation_Guide.rst

```diff
@@ -0,0 +1,610 @@
+|                         |                                                         |                                                                                                 | time, set up an *index pattern* for   |
+|                         |                                                         |                                                                                                 | the ``logstash-*`` index, using the   |
+|                         |                                                         |                                                                                                 | ``@timestamp`` field as *Time Filter  |
+|                         |                                                         |                                                                                                 | field name*.                          |
++-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------+
+
+See :doc:`../architecture/cluster-services` for more about these services
+and their configuration, or review the host sites for these projects.
+
+.. _Kubernetes dashboard: https://github.com/kubernetes/dashboard
+.. _Grafana: https://grafana.com
+.. _Cerebro: https://github.com/lmenezes/cerebro
+.. _Kibana: https://www.elastic.co/products/kibana/
+
+################
+Installing Zenko
+################
+
+*********
+Get Ready
+*********
+
+1. Remaining in the MetalK8s virtual shell, change to the directory from which
+   you will deploy Zenko:
+
+   ::
+
+   $ cd
+
+2. Initialize Helm:
+
+   ::
+
+    (metal-k8s) [centos@node01 ~]$ helm init
+    Creating /home/centos/.helm
+    Creating /home/centos/.helm/repository
+    Creating /home/centos/.helm/repository/cache
+    Creating /home/centos/.helm/repository/local
+    Creating /home/centos/.helm/plugins
+    Creating /home/centos/.helm/starters
+    Creating /home/centos/.helm/cache/archive
+    Creating /home/centos/.helm/repository/repositories.yaml
+    Adding stable repo with URL: https://kubernetes-charts.storage.googleapis.com
+    Adding local repo with URL: http://127.0.0.1:8879/charts
+    $HELM_HOME has been configured at /home/centos/.helm.
+    Warning: Tiller is already installed in the cluster.
+    (Use --client-only to suppress this message, or --upgrade to upgrade Tiller to the current version.)
+    Happy Helming!
+    (metal-k8s) [centos@node01 ~]$
+
+   Helm can now install applications on the Kubernetes cluster.
+
+3. Declare the ZooKeeper repository:
+
+   ::
+
+      $ helm repo add zenko-zookeeper https://scality.github.io/zenko-zookeeper/charts
+      "zenko-zookeeper" has been added to your repositories
+
+      $ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
```

Comment:
While these are no longer needed we need the following step `helm repo add scality https://scality.github.io/charts/`